roles/dtc/common/templates/ndfc_interfaces/ndfc_interface_access.j2

```diff
@@ -21,7 +21,7 @@
   profile:
     admin_state: {{ interface['enabled'] | default(defaults.vxlan.topology.switches.interfaces.topology_switch_access_interface.enabled) }}
     mode: 'access'
-    description: {{ interface['description'] | default(defaults.vxlan.topology.switches.interfaces.topology_switch_access_interface.description) }}
+    description: "{{ interface['description'] | default(defaults.vxlan.topology.switches.interfaces.topology_switch_access_interface.description) }}"
```

Comment:
The update to descriptions in this PR is to handle special characters that might appear in the description.